oci-pipeline-examples/oci-cascaded-pipelines/same-region/trigger-build-from-build/README.md

```diff
@@ -0,0 +1,59 @@
+# OCI DevOps Cascaded Pipelines
+
+## Trigger Build Pipeline from Build Pipeline
+
+### Prerequisites
+* Two build pipelines
+
+#### 1. Deploy Function
+**Step 1:**
+Clone the repository code.
+```
+$ git clone https://github.com/oracle-cloud-infra/oci-devops-cascaded-pipelines.git
+```
```

Comment:
Its not an oracle git repo ,so make it as a sparse checkout so that they can use the same repo for their functions